gtsam/hybrid/HybridFactorGraph.h

```diff
@@ -227,16 +151,13 @@ class HybridFactorGraph : protected FactorGraph<Factor>,
   /**
    * Simply prints the factor graph.
    */
-  void print(
+  virtual void print(
```

Comment:
Why does it not override?

Reply:
I further override it in the subclasses but I guess it's not required.